integration_tests/tests/s3.rs

```diff
@@ -21,6 +21,33 @@ use rusoto_core::default_tls_client;
 
 type TestClient = S3Client<DefaultCredentialsProvider, Client>;
 
+#[test]
+fn test_presigned_url() {
+    let _ = env_logger::init();
+
+    let client = S3Client::new(default_tls_client().unwrap(),
+                               DefaultCredentialsProvider::new().unwrap(),
+                               Region::EuCentral1);
+
+    let test_bucket = format!("rusoto_test_bucket_{}", get_time().sec);
+    let filename = format!("test_file_{}", get_time().sec);
+
+    test_get_presigned_url(&client, &test_bucket, &filename);
+}
+
+fn test_get_presigned_url(client: &TestClient, _bucket: &str, _filename: &str) {
+    let get_req = GetObjectRequest {
+        bucket: "store2be-papers-templates".to_string(),
+        key: "merge.tex.tera".to_string(),
+        ..Default::default()
+    };
+
+    let result = client.presigned_url(&get_req).unwrap();
+    println!("get object result: {:#?}", result);
+    assert_eq!(result, "tomato");
```

Comment:
This is failing on purpose, since I used it for testing a real s3 request and it doesn't make sense to hardcode our access key in there. This test could be made clean though.